packages/grid/x-data-grid/src/components/GridRow.tsx

```diff
@@ -57,6 +57,8 @@ export interface GridRowProps extends React.HTMLAttributes<HTMLDivElement> {
   tabbableCell: string | null;
   row?: GridRowModel;
   isLastVisible?: boolean;
+  indexOfColumnWithFocusedCellNotInRange?: number;
+  isNotVisible?: boolean;
```

Comment:
Is there a way we can reduce the name of `indexOfColumnWithFocusedCellNotInRange` a bit?

Reply:
what about this `focusedCellColumnIndexNotInRange`?

Reply:
Is the `NotInRange` strictly needed here?

Reply:
yes. If the focused cell is in range, it will be undefined. We need to find a way to indicate that the index is only held when it falls out of range.